templates/user_list.html

```diff
@@ -0,0 +1,64 @@
+{% extends "base-tw.html" %}
+
+{% load querystring_tools %}
+
+{% block metatitle %}Users | OpenSAFELY Jobs{% endblock metatitle %}
+
+{% block breadcrumbs %}
+  {% #breadcrumbs %}
+    {% url "home" as home_url %}
+    {% breadcrumb title="Home" url=home_url %}
+    {% breadcrumb title="Users" active=True %}
+  {% /breadcrumbs %}
+{% endblock breadcrumbs %}
+
+{% block content %}
+  <div class="
+    mb-6 mt-3 flex flex-col gap-y-4 md:gap-y-2 text-center
+    md:mt-0 md:text-left
+  ">
+    <h1 class="text-3xl tracking-tight break-words md:text-4xl font-bold text-slate-900">
+      Users
+    </h1>
+    <p class="font-lg text-slate-600">
+      Users using OpenSAFELY to deliver research projects.
+    </p>
+  </div>
+
+  {% #card %}
+    {% #list_group %}
+      {% for user in object_list %}
+        <li>
+          <a href="{{ user.get_absolute_url }}" class="block transition-colors duration-200 hover:bg-oxford-50">
+            <div class="flex items-center gap-x-3 px-4 py-3">
+              <div class="flex flex-col md:flex-row min-w-0 md:justify-between md:flex-grow">
+                <p class="truncate text-base font-semibold text-oxford-600">
+                  {{ user.name }}
+                </p>
+                <div class="mt-1 flex items-center text-sm text-slate-600 whitespace-nowrap md:mt-0 md:ml-2">
+                  {% icon_rectangle_stack_outline class="mr-1.5 h-5 w-5 flex-shrink-0 text-slate-400" %}
+                  {% if user.project_count %}
+                    {{ user.project_count }} project{{ user.project_count|pluralize }}
+                  {% else %}
+                    No projects
+                  {% endif %}
+                </div>
+              </div>
+            </div>
+          </a>
+        </li>
+      {% endfor %}
```

Comment:
This can use the list_group_item (although I should do some work to tidy this up).

```suggestion
      {% for user in object_list %}
        {% #list_group_item href=user.get_absolute_url %}
          <div class="flex flex-col md:flex-row min-w-0 md:justify-between md:flex-grow">
            <p class="truncate text-base font-semibold text-oxford-600">
              {{ user.name }}
            </p>
            <div class="mt-1 flex items-center text-sm text-slate-600 font-normal whitespace-nowrap md:mt-0 md:ml-2">
              {% icon_rectangle_stack_outline class="mr-1.5 h-5 w-5 flex-shrink-0 text-slate-400" %}
              {% if user.project_count %}
                {{ user.project_count }} project{{ user.project_count|pluralize }}
              {% else %}
                No projects
              {% endif %}
            </div>
          </div>
        {% /list_group_item %}
      {% endfor %}
```